core/templates/dev/head/components/summary_tile/ExplorationSummaryTileMobileDirective.js

```diff
@@ -0,0 +1,141 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for an exploration summary tile.
+ */
+
+oppia.directive('explorationSummaryTileMobile', [function() {
```

Comment:
I don't think the dimensions match with the library summary tiles, they look different in the mocks. Yeah, I'll try to figure out how to merge them into one directive.